<?xml version="1.0" encoding="UTF-8" ?>
<testsuite name="TestCafe Tests: Chrome 41.0.2227 / Mac OS X 10.10.1, Firefox 47 / Mac OS X 10.10.1" tests="6" failures="2" skipped="1" errors="2" time="925" timestamp="Thu, 01 Jan 1970 00:15:25 GMT" >
  <testcase classname="First fixture" file="./fixture1.js" name="�First test in first fixture (unstable) (screenshots: /screenshots/1445437598847)" time="74">
  </testcase>
  <testcase classname="First fixture" file="./fixture1.js" name="Second test in first fixture (screenshots: /screenshots/1445437598847)" time="74">
    <failure>
    <![CDATA[
      1) A JavaScript error occurred on "http://example.org".
         Repeat test actions in the browser and check the console for errors.
         Enable the “skipJsErrors” option to ignore JavaScript errors during test execution. Learn more: "https://testcafe.io/documentation/404038/recipes/debugging/skip-javascript-errors"
         If the website only throws this error when you test it with TestCafe, please create a new issue at:
         "https://github.com/DevExpress/testcafe/issues/new?template=bug-report.md".

         JavaScript error details:
         Some error

         Browser: Chrome 41.0.2227 / Mac OS X 10.10.1
         Screenshot: /screenshots/1445437598847/errors

            1 |var createCallsiteRecord = require('callsite-record');
            2 |
            3 |function someFunc () {
          > 4 |    throw new Error('Hey ya!');
            5 |}
            6 |
            7 |try {
            8 |    someFunc();
            9 |}

            at someFunc (some-file:1:1)
            at Object.<anonymous> (some-file:1:1)
            at Object.<anonymous> (some-file:1:1)
            at Object.<anonymous> (some-file:1:1)
            at Object.<anonymous> (some-file:1:1)
            at async formattedImport (some-file:1:1)
            at async Object.exports.requireOrImport (some-file:1:1)
            at async Object.exports.loadFilesAsync (some-file:1:1)
            at async singleRun (some-file:1:1)
            at async Object.exports.handler (some-file:1:1)

      2) The specified selector does not match any element in the DOM tree.

            | Selector
            |   one
          > |   two
            |   three

         Browser: Firefox 47 / Mac OS X 10.10.1

            1 |var createCallsiteRecord = require('callsite-record');
            2 |
            3 |function someFunc () {
          > 4 |    throw new Error('Hey ya!');
            5 |}
            6 |
            7 |try {
            8 |    someFunc();
            9 |}

            at someFunc (some-file:1:1)
            at Object.<anonymous> (some-file:1:1)
            at Object.<anonymous> (some-file:1:1)
            at Object.<anonymous> (some-file:1:1)
            at Object.<anonymous> (some-file:1:1)
            at async formattedImport (some-file:1:1)
            at async Object.exports.requireOrImport (some-file:1:1)
            at async Object.exports.loadFilesAsync (some-file:1:1)
            at async singleRun (some-file:1:1)
            at async Object.exports.handler (some-file:1:1)
    ]]>
    </failure>
  </testcase>
  <testcase classname="First fixture" file="./fixture1.js" name="Third test in first fixture" time="74">
  </testcase>
  <testcase classname="Second fixture" file="./fixture2.js" name="First test in second fixture" time="74">
  </testcase>
  <testcase classname="Second fixture" file="./fixture2.js" name="Second test in second fixture" time="74">
  </testcase>
  <testcase classname="Second fixture" file="./fixture2.js" name="Third test in second fixture" time="0">
    <skipped/>
  </testcase>
  <testcase classname="Third fixture" file="./fixture3.js" name="First test in third fixture (unstable)" time="74">
    <failure>
    <![CDATA[
      1) - Error in testRun.before hook -
         The specified selector does not match any element in the DOM tree.

            | Selector
            |   one
          > |   two
            |   three

         Browser: Firefox 47 / Mac OS X 10.10.1

            1 |var createCallsiteRecord = require('callsite-record');
            2 |
            3 |function someFunc () {
          > 4 |    throw new Error('Hey ya!');
            5 |}
            6 |
            7 |try {
            8 |    someFunc();
            9 |}

            at someFunc (some-file:1:1)
            at Object.<anonymous> (some-file:1:1)
            at Object.<anonymous> (some-file:1:1)
            at Object.<anonymous> (some-file:1:1)
            at Object.<anonymous> (some-file:1:1)
            at async formattedImport (some-file:1:1)
            at async Object.exports.requireOrImport (some-file:1:1)
            at async Object.exports.loadFilesAsync (some-file:1:1)
            at async singleRun (some-file:1:1)
            at async Object.exports.handler (some-file:1:1)
    ]]>
    </failure>
  </testcase>
  <system-out>
  <![CDATA[
    Warnings (3):
    --
      Was unable to take a screenshot due to an error.

      ReferenceError: someVar is not defined
    --
      Was unable to take a screenshot due to an error.

      ReferenceError: someOtherVar is not defined
    --
      Was unable to take screenshots because the screenshot directory is not specified. To specify it, use the "-s" or "--screenshots" command line option or the "screenshots" method of the test runner in case you are using API.
  ]]>
  </system-out>
</testsuite>
